shell/platform/android/android_context_gl.cc

```diff
@@ -206,12 +206,19 @@ std::unique_ptr<AndroidEGLSurface> AndroidContextGL::CreateOnscreenSurface(
     fml::RefPtr<AndroidNativeWindow> window) const {
   EGLDisplay display = environment_->Display();
 
-  const EGLint attribs[] = {EGL_NONE};
-
-  EGLSurface surface = eglCreateWindowSurface(
-      display, config_, reinterpret_cast<EGLNativeWindowType>(window->handle()),
-      attribs);
-  return std::make_unique<AndroidEGLSurface>(surface, display, context_);
+  if (window->IsOffscreen()) {
+    const EGLint attribs[] = {EGL_WIDTH, 1, EGL_HEIGHT, 1, EGL_NONE};
+
+    EGLSurface surface = eglCreatePbufferSurface(display, config_, attribs);
```

Comment:
Call `AndroidContextGL::CreatePbufferSurface` which is identical to this code.

Reply:
Done.